frontend/src/pages/AccountSend.vue

```diff
@@ -476,6 +465,13 @@
                 type="submit"
                 :disable="batchSends.length == 1 || isSending"
               />
+              <div class="q-mt-sm">
+                <router-link :class="{ 'no-text-decoration': true, 'dark-toggle': true }" :to="{ name: 'sent' }">
+                  <div class="row items-center justify-center q-pa-sm link-container rounded-borders">
+                    {{ $t('Send.send-history') }}
+                  </div>
+                </router-link>
+              </div>
```

Comment:
was this element changed or just moved?

Reply:
Yes, it was feature flagged to be hidden so it wasn't showing up on the frontend. I moved it to be below the send button like we have it on the regular send page, but also happy to move if that looks weird.